src/helm/helmfile.yaml

```diff
@@ -32,7 +36,8 @@ repositories:
 
 releases:
   - name: postgres
-    installed: {{ eq .Environment.Name "dev" | toYaml }}
+    installed: {{ or (eq .Environment.Name "dev") (eq .Environment.Name "dev-keycloak") | toYaml }}
+    missingFileHandler: Warn
```

Comment:
Out of curiosity, why do we need to add `missingFileHandler` ?

Reply:
It's because this file env.d/{{ .Environment.Name }}/values.secrets.yaml exist only for the new environment